integration-tests/tests/common/mod.rs

```diff
@@ -790,10 +797,23 @@ impl<'a> TestServerConfigurator<'a> {
                     "config_dir",
                     &self.test_context.runner_path(&self.server_number),
                 );
-                cfg.insert(
-                    "user",
-                    users::get_current_username().unwrap().to_str().unwrap(),
-                );
+
+                if per_device == "false" {
```

Comment:
it makes more sense to me to get the value from the env var, convert it to boolean and then check this case as a boolean over here instead of comparing it to a string

Reply:
yes above m reading from env variable, added the part to compare with boolean instead of string